language/resource.language.fi_fi/strings.po

```diff
@@ -0,0 +1,2420 @@
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31214"
+msgid "Series Finale"
+msgstr "Sarjan päätösjakso"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31215"
+msgid "Season Finale"
+msgstr "Kauden päätösjakso"
+
+msgctxt "#31216"
+msgid "Video decoder"
+msgstr "Videodekooderi"
+
+msgctxt "#31217"
+msgid "Pixel format"
+msgstr "Pikselimuoto"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31218"
+msgid "Mid-Season Finale"
+msgstr "Puolivälin päätösjakso"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31219"
+msgid "Left header"
+msgstr "Vasen otsikkoteksti"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31220"
+msgid "Purple"
+msgstr "Violetti"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31221"
+msgid "Slate"
+msgstr "Litteä"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31222"
+msgid "Coal"
+msgstr "Hiili"
+
+msgctxt "#31223"
+msgid "Gridlines"
+msgstr "Ruudukko"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31224"
+msgid "Pink"
+msgstr "Vaalenpunainen"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31225"
+msgid "Chalk"
+msgstr "Kalkki"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31226"
+msgid "Blush"
+msgstr "Blush"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31227"
+msgid "Premiere"
+msgstr "Ensiesitys"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31228"
+msgid "Aqua"
+msgstr "Sininen"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31229"
+msgid "Vaporwave"
+msgstr "Vaporwave"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31230"
+msgid "Calendar"
+msgstr "Kalenteri"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31231"
+msgid "Customise highlight colours"
+msgstr "Korostus"
+
+#: /1080i/Custom_1111_Dialog_ColourPicker.xml
+msgctxt "#31232"
+msgid "Gradient"
+msgstr "Liukuväri"
+
+msgctxt "#31233"
+msgid "Seek to"
+msgstr "Siirry kohtaan"
+
+msgctxt "#31234"
+msgid "Customise indicators"
+msgstr "Muokkaa merkintöjä"
+
+msgctxt "#31235"
+msgid "Indicators"
+msgstr "Merkinnät"
+
+msgctxt "#31236"
+msgid "Collection"
+msgstr "Kokoelma"
+
+msgctxt "#31237"
+msgid "Progress"
+msgstr "Edistyminen"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31238"
+msgid "Coral"
+msgstr "Koralli"
+
+msgctxt "#31239"
+msgid "Footer"
+msgstr "Oikea alanurkka"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31240"
+msgid "Tropical"
+msgstr "Trooppinen"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31241"
+msgid "Colour scheme"
+msgstr "Väriteema"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31242"
+msgid "Blue Slate"
+msgstr "Sininen"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31243"
+msgid "Midnight Purple"
+msgstr "Keskiyön violetti"
+
+msgctxt "#31244"
+msgid "Now playing"
+msgstr "Nyt toistetaan"
+
+msgctxt "#31245"
+msgid "Autoscroll labels"
+msgstr "Tekstirivin automaattinen vieritys"
+
+msgctxt "#31246"
+msgid "Autoscroll textboxes"
+msgstr "Tekstikappaleen automaattinen vieritys"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31247"
+msgid "Colour highlight"
+msgstr "Värillinen korostus"
+
+msgctxt "#31248"
+msgid "Building includes"
+msgstr "Rakennetaan valikoita"
```

Comment:
"includes" should be left untranslated as it refers to the `Includes.xml` skin files